dist/Lob-API-bundled.yml

```diff
@@ -526,7 +526,7 @@ components:
           description: >
             Geographic direction preceding a street name (`N`, `S`, `E`, `W`,
             `NE`, `SW`, `SE`, `NW`).
-          example: ''
+          example: 'N'
```

Comment:
I believe this example will show up in the example address given (the 185 BERRY ST one), which doesn't actually have any directionals IRL. Should we change the address to one which does have pre- and post-directionals, which would be a break from the current docs, or is it okay to put in a directional where there isn't one?

Reply:
Ok, so I looked at the documentation again, and it appears to me as if what we need is to include the empty string in the enum list, as both pre and post directionals are required.